src/util/common-util.ts

```diff
@@ -6,12 +6,16 @@ import { Address, BaseAddressModel } from '@/models/Address'
 import { Role, SlipStatus } from '@/util/constants'
 
 import KeyCloakService from 'sbc-common-components/src/services/keycloak.services'
-import moment from 'moment'
+import * as moment from 'moment-timezone'
 
 export default class CommonUtils {
   // Formatting date in the desired format for displaying in the template
   static formatDisplayDate (date: Date, format?: string) {
-    return date ? moment(date).format(format || 'MMM DD, YYYY') : ''
+    if (date) {
+      return moment.tz(date, 'Canada/Vancouver').format(format || 'MMM DD, YYYY')
```

Comment:
Should be America/Vancouver

Reply:
idk why but weirdly America/Vancouver doesn't seem to convert dates to PST while Canada/Vancouver does
made some routing slips with a cheque date on December 1st 2022 and November 9th 2022

America/Vancouver
![image](https://user-images.githubusercontent.com/82549206/205144267-bc3f1d15-fb7b-493d-8aa0-5a6f9131bcb0.png)

![image](https://user-images.githubusercontent.com/82549206/205144233-142c547c-4420-450f-bfad-b18a6131e136.png)

Canada/Vancouver
![image](https://user-images.githubusercontent.com/82549206/205144747-1e356e97-3487-4c42-90a3-c55f47ec9552.png)

![image](https://user-images.githubusercontent.com/82549206/205144832-77dd4247-8e8d-44d5-95db-e7e032646681.png)

Reply:
nvm the cheque dates have UTC times set to 00:00:00 -> -8h to convert to pst -> 16:00:00 the day before
probably best to change it so that UTC times are set to 08:00:00 so that conversion keeps it on the same day